packages/fetchai/contracts/erc1155/contract.yaml

```diff
@@ -16,3 +16,4 @@ description: "The erc1155 contract implements an ERC1155 contract package."
 path_to_contract_interface: "build/erc1155.json"
 dependencies:
   vyper: { version: "==0.1.0b12"}
+class_name: ERC1155Contract
```

Comment:
Not sure why we are adding this, we should use the `*Contract` convention like for the other components. If we do introduce a change to this it should be consistent.

Reply:
for connections, we have the class name.